site/docs/v1/tech/oauth/url-validation.adoc

```diff
@@ -0,0 +1,274 @@
+---
+layout: doc
+title: URL Validation
+description: A description OAuth URL validation policies in FusionAuth
+navcategory: login-methods
+---
+
+[NOTE.since]
+====
+Available since 1.43.0
+====
+
+== Overview
+
+Beginning in version `1.43.0` FusionAuth provides support for wildcards in OAuth 2.0 redirect URLs and origin URLs. This document provides details on where wildcards are allowed in configured values and the valid replacement patterns for wildcards in each position.
+
+* <<URL Validation Policy>>
+* <<Allowed Wildcard Positions>>
+** <<Domain>>
+** <<Port>>
+** <<Path Segments>>
+** <<Query String Values>>
+* <<Wildcard Replacement Patterns>>
+** <<Domain>>
+** <<Port>>
+** <<Path Segments>>
+** <<Query String Values>>
+
+== URL Validation Policy
+
+In order to configured authorized redirect and origin URLs with wildcards, the [field]#URL validation# setting must be configured to `Allow wildcards` under [breadcrumb]#Applications -> Edit Application -> OAuth#.
+
+See the link:/docs/v1/tech/apis/applications[Application API] or [Application OAuth Configuration](/docs/v1/tech/core-concepts/applications#oauth) for details.
+
+== Allowed Wildcard Positions
+
+In order to maintain security while allowing the flexibility of wildcards, FusionAuth limits the position and number of wildcards that are allowed in the configured authorized redirect and origin URLs. The `*` is used to denote a wildcard.
+
+=== Domain
+
+The domain of a configured URL allows 0 or 1 wildcards in the domain portion of the URL. If the domain contains a wildcard, it must meet all of the following requirements:
+
+ * The domain must contain at least three segments
+ * The wildcard may only appear in the host portion of the domain (left-most subdomain)
+ * The wildcard can be a full or partial replacement of the host
+
+The following table provides some examples of valid and invalid wildcard patterns.
+
+[cols="45m,10,45"]
+.Domain Wildcards
+|===
```

Comment:
We have a similar table here https://site-local.fusionauth.io/docs/v1/tech/guides/breached-password-detection#match-mode in `Table 1. What Is Allowed In Each Match Mode`, we should be able to copy the way we did check marks and x's.